docs/src/main/sphinx/release/release-382.md

```diff
@@ -0,0 +1,58 @@
+# Release 382 (25 May 2022)
+
+## General
+
+* Add support for fault-tolerant execution with [exchange spooling on Google Cloud Storage](fte-exchange-manager). ({issue}`12360`)
+* Drop support for exchange spooling on S3 with for the legacy schemes `s3n://` and `s3a://`. ({issue}`12360`)
+* Improve join performance. ({issue}`12257`)
+* Improve coordinator memory management for task-based fault-tolerant execution.
+  This decreases the chance of queries failing with the
+  `EXCEEDED_TASK_DESCRIPTOR_STORAGE_CAPACITY` error code. ({issue}`12478`)
+
+## BigQuery connector
+
+* Add support for [using cached query results of BigQuery](https://cloud.google.com/bigquery/docs/cached-results).
```

Comment:
What's the impact? Which kinds of queries would benefit?

Reply:
there is a lot of details to that and they are specific to BigQuery .. the documentation on the Google site goes into that and hence we just link there.

Reply:
Users can reduce their cost and get result faster. The cache will be reused when the generated SQL matches the past SQL.